static/js/map/map.pokemon.js

```diff
@@ -268,6 +274,17 @@ function pokemonLabel(item) {
                 </div>`
         }
 
+        
+        var sizeDisplay = ''
```

Comment:
Can you move this variable declaration up to the others?  (next to var statsDisplay = '' and friends)